### ISSUE TITLE
Gnome 47

### ISSUE THREAD

**Issue by washran** [penguin-teal/gnome-openweather#74]
## Description

Upgraded to gnome 47.

## System

Ubuntu 24.10

## Settings

```json
Extension disabled. System says incompatible with gnome 47.
```

**Comment by penguin-teal** [comment]
Since there was a feature/API freeze I will start working on it however I'll only try to have it out by the time GNOME 47 actually releases.

**Comment by creeper82** [comment]
Any progress? Of course take your time, but gnome without your extension feels really empty.. :c

**Comment by dsheeler** [comment]
@creeper82, I just tried adding 47 to the supported versions in metadata.json and it seems to work alright.

**Comment by washran** [comment]
I switched to Archlinux. I added 47 to metadata.json like creeper82 did. 

metadata.json works. pasting contents here.

{"app-version":"138","git-version":"138.r0.g6367a11","gnome-version":"47.0","user-locale":"en","unit":"'fahrenheit'","custom-keys":["<REDACTED>","",""],"appid":"<REDACTED>","weather-provider":"'openweathermap'","has-run":"true","days-forecast":"4","refresh-interval-current":"1200","delay-ext-init":"5","locs":[{}],"show-sunsetrise-in-panel":"true","my-loc-prov":"'ipinfoio'","weatherapidotcom-key":"<REDACTED>","position-index":"1","geolocation-appid-mapquest":"<REDACTED>","pressure-unit":"'inHg'","menu-alignment":"47.0","position-in-panel":"'center'","wind-speed-unit":"'mph'","show-comment-in-panel":"true"}
